File: languageExtensions/OrchidAsciidoc/src/main/java/com/eden/orchid/languages/asciidoc/AsciidocModule.java
package com.eden.orchid.languages.asciidoc;

import com.eden.orchid.api.compilers.OrchidCompiler;
import com.eden.orchid.api.registration.OrchidModule;

public class AsciidocModule extends OrchidModule {

    @Override
    protected void configure() {
        // Compilers
        addToSet(OrchidCompiler.class,
                AsciiDoctorCompiler.class);
    }
}

File: languageExtensions/OrchidAsciidoc/src/main/kotlin/com/eden/orchid/languages/asciidoc/AsciidocModule.kt
package com.eden.orchid.languages.asciidoc

import com.eden.orchid.api.compilers.OrchidCompiler
import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.utilities.addToSet

class AsciidocModule : OrchidModule() {

    override fun configure() {
        addToSet<OrchidCompiler, AsciiDoctorCompiler>()
    }
}
